NorthstarDLL/plugins/plugins.cpp

```diff
@@ -188,8 +188,36 @@ std::optional<Plugin> PluginManager::LoadPlugin(fs::path path, PluginInitFuncs*
 	return plugin;
 }
 
+inline void FindPlugins(fs::path pluginPath, std::vector<fs::path>& paths)
+{
+	// ensure dirs exist
+	if (!fs::exists(pluginPath) || !fs::is_directory(pluginPath))
+	{
+		return;
+	}
+
+	fs::recursive_directory_iterator iterator(pluginPath);
```

Comment:
Ugh, I just realised another issue with this. 

This doesn't do the regex check that the mod-manager does

https://github.com/R2Northstar/NorthstarLauncher/blob/82bff57d166ece1173ca7df33940171548015a71/NorthstarDLL/mods/modmanager.cpp#L622-L623

Also I'm pretty sure it still loads plugins recursively which we shouldn't need to do anymore. The only reason we added recursive load was to have a plugin -> Thunderstore-package mapping with the old plugin system, i.e. #460 which can also be reverted now.